cfg/dancer.conf

```diff
@@ -1,10 +1,18 @@
-PerlPassEnv MYSQL_USER
-PerlPassEnv MYSQL_PASSWORD
-PerlPassEnv MYSQL_DATABASE
-PerlPassEnv MYSQL_ROOT_PASSWORD
-
 <Location />
   SetHandler perl-script
   PerlResponseHandler Plack::Handler::Apache2
   PerlSetVar psgi_app /opt/openshift/src/bin/app.psgi
 </Location>
+
+
+<Perl>
+local $ENV{PATH} = '/usr/bin';
+local $_;
+
+for (`env`) {
+    $_=~s/(.*?)=.*/$1/;
+    push @PassEnv, $_;
+}
+</Perl>
```

Comment:
@bparees this PassEnv thing was a big pain in v2 world... I remember that we generated 'passenv.conf' for httpd when we start the gear. I wonder if our httpd images should not do the same.

We will have same issue in PHP where the variable will not available in PHP as the httpd will not pass it...